src/main/java/spoon/support/reflect/code/CtTypeAccessImpl.java

```diff
@@ -45,7 +45,7 @@ public void accept(CtVisitor visitor) {
 
 	@Override
 	public CtTypeReference<Void> getType() {
-		return (CtTypeReference<Void>) getFactory().Type().VOID_PRIMITIVE.clone().<CtTypeAccess>setParent(this);
+		return getFactory().Type().VOID_PRIMITIVE.clone().setParent(this);
```

Comment:
Case and point in small amount of breakage: this was the only part of the entire Spoon codebase that needed to change, apart from the directly affected method headers.